nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-json-record-utils/src/main/java/org/apache/nifi/json/AbstractJsonRowRecordReader.java

```diff
@@ -51,16 +53,31 @@
 import java.util.function.Supplier;
 
 public abstract class AbstractJsonRowRecordReader implements RecordReader {
+    static final PropertyDescriptor MAX_STRING_LENGTH = new PropertyDescriptor.Builder()
+            .name("max-string-length")
```

Comment:
Recommend starting out with the same value for name.
```suggestion
            .name("Max Strength Length")
```

Reply:
I'd copied for consistency this from existing updates to Processors such as `SplitJson`, but I've no issues renaming it here if we're OK to accept the inconsistency (only likely a frustration for anyone scripting processor/controller updates rather than using the NiFi UI)

Reply:
Happy to update the existing properties, which will impact (unreleased) `EvaluateJsonPath`, `SplitJson`, `JoltTransformJSON` processors

Reply:
Thanks, I would support making that change. Do you want to do that in this pull request, or separately? Whatever you prefer sounds good.

Reply:
Great, thanks!